Formula/tiv.rb

```diff
@@ -15,10 +15,7 @@ class Tiv < Formula
   fails_with gcc: "5"
 
   def install
-    cd "src/main/cpp" do
-      system "make"
-      bin.install "tiv"
-    end
+    system "make", "-C", "src/main/cpp", "prefix=#{prefix}", "install"
```

Comment:
```suggestion
    system "make", "-C", "src/main/cpp", "prefix=#{prefix}", "INSTALL_PROGRAM=install -d", "install"
```
BSD `install` doesn't like `-D`.

Reply:
ugh.

Reply:
"tiv exists but is not a directory"

Reply:
Oops. Yea, this is more trouble that it's worth. We can go back to your original `install` method. Sorry for leading you down a dead end!